src/pages/Login/style.css

```diff
@@ -0,0 +1,120 @@
+@import url('../../styles/variables.css');
```

Comment:
You don't need to import variables.css, since it's already being imported in App.js you can access it inside this file